api/raw.go

```diff
@@ -0,0 +1,41 @@
+package api
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/TwiN/gatus/v5/storage/store"
+	"github.com/TwiN/gatus/v5/storage/store/common"
+	"github.com/gofiber/fiber/v2"
+)
+
+func UptimeRaw(c *fiber.Ctx) error {
+	duration := c.Params("duration")
+	var from time.Time
+	switch duration {
```

Comment:
It should be possible to support 30d as well now. Sorry, I had missed the notification.

Reply:
@TwiN Updated to support 30d uptime 😄